prody/proteins/interactions.py

```diff
@@ -41,10 +41,10 @@
            'calcPiCationTrajectory', 'calcHydrophobicTrajectory', 'calcDisulfideBondsTrajectory',
            'calcProteinInteractions', 'calcStatisticsInteractions', 'calcDistribution',
            'calcSASA', 'calcVolume','compareInteractions', 'showInteractionsGraph',
-           'calcLigandInteractions', 'listLigandInteractions', 
+           'calcLigandInteractions', 'showLigandInteractions', 
```

Comment:
Ok. I will change that. We have never used listSOMETHING (more often show), so I thought it would be better.

Reply:
thanks. I think we haven't really done much listing of things so we haven't needed to have listSomething. I do get that users are more likely to find showSomething because they expect it but they may be surprised by the results

Reply:
Done